tools/cpp/unix_cc_toolchain_config.bzl

```diff
@@ -761,7 +761,6 @@ def _impl(ctx):
             ),
         ],
         requires = [feature_set(features = ["coverage"])],
-        provides = ["profile"],
```

Comment:
This matches the macOS toolchain which uses similar features, since this requires coverage I think that's safe, since coverage provides profile